modules/ROOT/pages/kafka/kafka-connector-reference.adoc

```diff
@@ -1,25 +1,30 @@
-= Apache Kafka Module Documentation Reference
+= Apache Kafka Connector Reference
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+Support Category: Select
+
+Apache Kafka Connector v3.0
+
+
 == Configurations
 ---
 [[kafka-consumer-config]]
 === Consumer configuration
 
 
 ==== Parameters
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
 |Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
-| Connection a| * <<kafka-consumer-config_basic-kafka-consumer-connection, Kafka Basic Consumer Connection>>
-* <<kafka-consumer-config_kerberos-kafka-consumer-connection, Kafka Kerberos Consumer Connection>>
-* <<kafka-consumer-config_kerberos-ssl-kafka-consumer-connection, Kafka Kerbero SSL Consumer Connection>>
-* <<kafka-consumer-config_ssl-kafka-consumer-connection, Kafka SSL Consumer Connection>>
- | The connection types that can be provided to this configuration. | | x
-| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform will expire the instance at the exact moment that it becomes eligible. The runtime will actually purge the instances when it sees it fit. |  |
+| Connection a| * <<kafka-consumer-config_basic-kafka-consumer-connection, Kafka Basic Consumer Connection>> 
+* <<kafka-consumer-config_kerberos-kafka-consumer-connection, Kafka Kerberos Consumer Connection>> 
+* <<kafka-consumer-config_kerberos-ssl-kafka-consumer-connection, Kafka Kerbero SSL Consumer Connection>> 
+* <<kafka-consumer-config_ssl-kafka-consumer-connection, Kafka SSL Consumer Connection>> 
+ | The connection types to provide to this configuration. | | x
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime purges the instances when it sees fit. |  | 
```

Comment:
The runtime purges the instances when it sees fit. 

"when it sees fit" isn't translation friendly. And what runtime? Mule? What about something like: 
The instance might be purged at a later time. 
or
Mule will purge the instance when it is appropriate. 
or 
Mule might choose to purge the instance at a later time. 

It's a tough one...

Reply:
Went with:
Mule purges the instances when it is appropriate.